libmamba/src/core/transaction.cpp

```diff
@@ -921,7 +921,7 @@ namespace mamba
             return true;
         }
 
-        auto lf = LockFile::create_lock(ctx.target_prefix / "conda-meta");
```

Comment:
this also changes behavior, but it is OK (I need to do the same in the self-update PR).